css/style.css

```diff
@@ -0,0 +1,1027 @@
+}
+
+@media (min-width: 796px) {
+  .wrappersubgrid-seven {
+    margin-top: 5em;
+    display: grid;
+    width: 100%;
+    grid-template-columns: 1fr 1fr 1fr 1fr;
+    grid-template-areas: "first-case second-case thrid-case fourth-case" "fifth-case sixth-case seventh-case eighth-case";
+  }
+}
+
+@media (max-width: 795px) and (min-width: 600px) {
+  .wrappersubgrid-seven {
+    display: grid;
+    width: 100%;
+    grid-template-columns: 1fr 1fr;
+    grid-template-areas: "first-case second-case" "thrid-case fourth-case" "fifth-case sixth-case" "seventh-case eighth-case";
+  }
+}
+
+.cover-trigger {
+  background-color: rgba(0, 0, 0, 0.25) url("https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/image-gradient.png") no-repeat;
+}
+
+@media (min-width: 797px) {
+  .cover-trigger {
+    background-color: rgba(0, 0, 0, 0.25) url("https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/image-gradient.png") no-repeat;
+    margin-left: 0.5em;
+  }
+}
+
+footer div.group-link div ul > li > a:hover {
+  margin-left: 3px;
+  margin-right: 3px;
+  color: white;
+  text-decoration: none;
+}
+
+footer div.group-link div ul > li {
+  margin-bottom: 0;
+  display: inline-block;
+}
+
+footer div.group-link {
+  background: #131313;
+  padding-top: 3em;
+  padding-bottom: 3em;
+}
+
+ul.autor-time > li:hover {
+  display: inline-block;
+  padding-left: 0;
+  color: black;
+}
+
+footer div.group-link > div:nth-child(1) > ul > li {
+  padding-left: 0.5em;
+  padding-right: 0.5em;
+}
+
+footer div.group-link > div:nth-child(2) > ul {
+  margin-top: 2em;
+  font-size: 12px;
+}
+
+p.copyright {
+  margin-top: 0;
+  margin-bottom: 0;
+  background-color: #000;
+  color: #aaa;
+  font-size: 13px;
+  padding-top: 3em;
+  padding-bottom: 3em;
+  text-align: center;
+}
+
+@media (min-width: 768px) {
+  .textdescription-group > h2 {
+    font-size: 28px;
+  }
+}
+
+.second-content .wrappersubgrid-two > .first-case > img,
+.second-content .wrappersubgrid-two > .second-case > img,
+.second-content .wrappersubgrid-two > .third-case > img,
+.second-content .wrappersubgrid-two > .fourth-case > img,
+.second-content .wrappersubgrid-two > .fifth-case > img,
+.second-content .wrappersubgrid-two > .sixth-case > img,
+.second-content .wrappersubgrid-two > .seventh-case > img,
+.second-content .wrappersubgrid-two > .eighth-case > img {
+  height: 160px;
+  width: 96%;
+}
+
+.second-content .wrappersubgrid-two > .first-case > h3,
+.second-content .wrappersubgrid-two > .second-case > h3,
+.second-content .wrappersubgrid-two > .third-case > h3,
+.second-content .wrappersubgrid-two > .fouth-case > h3,
+.second-content .wrappersubgrid-two > .fifth-case > h3,
+.second-content .wrappersubgrid-two > .sixth-case > h3,
+.second-content .wrappersubgrid-two > .seventh-case > h3,
+.second-content .wrappersubgrid-two > .eighth-case > h3 {
+  text-align: initial;
```

Comment:
Let's use class instead of long selectors.

Reply:
ok, thanks. I changed the selectores